diff -up Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/demos/CMakeLists.txt.smoke Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/demos/CMakeLists.txt
--- Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/demos/CMakeLists.txt.smoke	2018-05-08 10:00:26.738964106 +1000
+++ Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/demos/CMakeLists.txt	2018-05-08 10:01:39.240848209 +1000
@@ -173,54 +173,3 @@ if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux"
     endif()
 endif()
 
-######################################################################################
-# cube
-
-if(APPLE)
-    include(macOS/cube/cube.cmake)
-elseif(NOT WIN32)
-    if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL ${CMAKE_HOST_SYSTEM_PROCESSOR})
-        add_executable(cube cube.c ${PROJECT_SOURCE_DIR}/demos/cube.vert ${PROJECT_SOURCE_DIR}/demos/cube.frag cube.vert.inc cube.frag.inc)
-        target_link_libraries(cube ${LIBRARIES})
-    endif()
-else()
-    if (CMAKE_CL_64)
-        set (LIB_DIR "Win64")
-    else()
-        set (LIB_DIR "Win32")
-    endif()
-
-    add_executable(cube WIN32 cube.c ${PROJECT_SOURCE_DIR}/demos/cube.vert ${PROJECT_SOURCE_DIR}/demos/cube.frag cube.vert.inc cube.frag.inc)
-    target_link_libraries(cube ${LIBRARIES})
-endif()
-
-######################################################################################
-# cubepp
-
-if(APPLE)
-    include(macOS/cubepp/cubepp.cmake)
-elseif(NOT WIN32)
-    if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL ${CMAKE_HOST_SYSTEM_PROCESSOR})
-        add_executable(cubepp cube.cpp ${PROJECT_SOURCE_DIR}/demos/cube.vert ${PROJECT_SOURCE_DIR}/demos/cube.frag cube.vert.inc cube.frag.inc)
-        target_link_libraries(cubepp ${LIBRARIES})
-    endif()
-else()
-    if (CMAKE_CL_64)
-        set (LIB_DIR "Win64")
-    else()
-        set (LIB_DIR "Win32")
-    endif()
-
-    add_executable(cubepp WIN32 cube.cpp ${PROJECT_SOURCE_DIR}/demos/cube.vert ${PROJECT_SOURCE_DIR}/demos/cube.frag cube.vert.inc cube.frag.inc)
-    target_link_libraries(cubepp ${LIBRARIES})
-endif()
-
-######################################################################################
-# smoke
-
-if ((${CMAKE_SYSTEM_PROCESSOR} STREQUAL ${CMAKE_HOST_SYSTEM_PROCESSOR}))
-    if ((DEMOS_WSI_SELECTION STREQUAL "XCB") OR (DEMOS_WSI_SELECTION STREQUAL "WAYLAND") OR WIN32 OR
-        (CMAKE_SYSTEM_NAME STREQUAL "Android") OR (CMAKE_SYSTEM_NAME STREQUAL "Darwin"))
-        add_subdirectory(smoke)
-    endif()
-endif()
