diff -up Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/CMakeLists.txt.da Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/CMakeLists.txt
--- Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/CMakeLists.txt.da	2018-04-20 08:07:56.000000000 +1000
+++ Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/CMakeLists.txt	2018-05-08 07:50:10.179958598 +1000
@@ -17,7 +17,6 @@ set(MAJOR "1")
 string(TOLOWER ${API_NAME} API_LOWERCASE)
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-find_package(PythonInterp 3 REQUIRED)
 
 option(USE_CCACHE "Use ccache" OFF)
 if (USE_CCACHE)
diff -up Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/lvl_genvk.py.da Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/lvl_genvk.py
--- Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/lvl_genvk.py.da	2018-04-20 08:07:56.000000000 +1000
+++ Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/lvl_genvk.py	2018-05-08 07:50:10.179958598 +1000
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/python
 #
 # Copyright (c) 2013-2018 The Khronos Group Inc.
 #
diff -up Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/object_tracker_generator.py.da Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/object_tracker_generator.py
--- Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/object_tracker_generator.py.da	2018-04-20 08:07:56.000000000 +1000
+++ Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/object_tracker_generator.py	2018-05-08 07:50:10.180958624 +1000
@@ -263,7 +263,7 @@ class ObjectTrackerOutputGenerator(Outpu
             ]
         for vuid_filename in vuid_filename_locations:
             if os.path.isfile(vuid_filename):
-                self.vuid_file = open(vuid_filename, "r", encoding="utf8")
+                self.vuid_file = open(vuid_filename, "r")
                 break
         if self.vuid_file == None:
             print("Error: Could not find vk_validation_error_messages.h")
diff -up Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/parameter_validation_generator.py.da Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/parameter_validation_generator.py
--- Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/parameter_validation_generator.py.da	2018-04-20 08:07:56.000000000 +1000
+++ Vulkan-LoaderAndValidationLayers-sdk-1.1.73.0/scripts/parameter_validation_generator.py	2018-05-08 07:50:10.180958624 +1000
@@ -195,7 +195,7 @@ class ParameterValidationOutputGenerator
             ]
         for vuid_filename in vuid_filename_locations:
             if os.path.isfile(vuid_filename):
-                self.vuid_file = open(vuid_filename, "r", encoding="utf8")
+                self.vuid_file = open(vuid_filename, "r")
                 break
         if self.vuid_file == None:
             print("Error: Could not find vk_validation_error_messages.h")
