diff -up Vulkan-ValidationLayers-sdk-1.1.97.0/tests/layers/CMakeLists.txt.dma Vulkan-ValidationLayers-sdk-1.1.97.0/tests/layers/CMakeLists.txt
--- Vulkan-ValidationLayers-sdk-1.1.97.0/tests/layers/CMakeLists.txt.dma	2019-02-26 09:09:22.661039052 +1000
+++ Vulkan-ValidationLayers-sdk-1.1.97.0/tests/layers/CMakeLists.txt	2019-02-26 09:09:25.906125123 +1000
@@ -17,9 +17,6 @@
 
 set(TEST_LAYER_NAME VkLayer_device_profile_api)
 
-set(VK_LAYER_RPATH /usr/lib/x86_64-linux-gnu/vulkan/layer:/usr/lib/i386-linux-gnu/vulkan/layer)
-set(CMAKE_INSTALL_RPATH ${VK_LAYER_RPATH})
-
 if(WIN32)
     if(NOT (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_CURRENT_BINARY_DIR))
         file(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/windows/${TEST_LAYER_NAME}.json src_json)
